composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "sonata-project/media-orm-pack",
+    "type": "symfony-pack",
+    "description": "A pack for SonataMediaBundle with ORM support",
+    "keywords": [
+        "media",
+        "sonata",
+        "orm"
+    ],
+    "homepage": "https://sonata-project.org/bundles/media",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Thomas Rabaix",
+            "email": "thomas.rabaix@sonata-project.org",
+            "homepage": "https://sonata-project.org"
+        },
+        {
+            "name": "Sonata Community",
+            "homepage": "https://github.com/sonata-project/media-orm-pack/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.0",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.0",
+        "sonata-project/media-bundle": "^3.13"
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "3.x-dev"
```

Comment:
Should we even have this?

Reply:
@kunicmarko20 you are right, i left it for a case if it is needed. Now composer.json contains only `require` section and some meta data